providers/azurecompute-arm/src/test/java/org/jclouds/azurecompute/arm/features/VirtualMachineApiMockTest.java

```diff
@@ -119,6 +119,22 @@ public void testListEmpty() throws Exception {
             + "/virtualMachines?api-version=2018-06-01");
    }
 
+   public void testListAll() throws Exception {
+      server.enqueue(jsonResponse("/virtualmachinesinsubscription.json"));
+      final VirtualMachineApi vmAPI = api.getVirtualMachineApi(null);
+      assertEquals(vmAPI.listAll(), getVMListAll()); //TODO allinsubs
```

Comment:
What does this TODO and the other one stand for? Can they be removed or is there anything else TBD?

Reply:
That one is already done, forgot to remove it!